dockstore-yaml-proposals/jbrowse/docker-compose.yml

```diff
@@ -8,3 +8,11 @@ services:
     - "${HTTP_PORT:-3000}:3000"
     volumes:
     - ./data:/jbrowse/data
+    deploy:
```

Comment:
What is jbrowse? Is it used anywhere in appstore? @n0mn0m @stevencox

Reply:
I am not sure what it is, but running a search for jbrowse across projects I do not see anything outside of this repo.

Reply:
@ismagilani @n0mn0m - let's leave xena and jbrowse. We share this repo with others and those are, I believe, created by others.